scripts/build_scenario1.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+from jinja2 import Environment, FileSystemLoader
+
+# Définir le répertoire contenant le template
+template_dir = '/home/chausser/test/'
+
+
+
+# Configuration de Jinja2 pour charger le template
+env = Environment(loader=FileSystemLoader(template_dir))
+template = env.get_template('scenario1.yaml.j2')
+for missing in range(1,7):
+    for width in range(1,9):
+        for depth in range(2,11):
+            name = 'scenario_1_depth'+str(depth)+'_width'+str(width)+'_missing'+str(missing)
+            # Les données à insérer dans le template
+            missing_rate = 0.1*missing
+            missing_rate = round(missing_rate,3)
+            data = {
+                'meta': {
+                    'language': 'en'
+                },
+                'variants': {
+                    'letters': {
+                        'mispell': {
+                            'law': 'Bernouilli',
+                            'rate': 0.5,
+                        }
+                    },
+                    'words': {
+                        'synonym': {
+                            'law': 'Bernouilli',
+                            'rate': 0
+                        },
+                        'omit': {
+                            'law': 'Bernouilli',
+                            'rate': 0.05
+                        }
+                    },
+                    'sentences': {
+                        'duplicate': {
+                            'args': {
+                                'nbr_words': 2
+                            },
+                            'law': 'Bernouilli',
+                            'rate': 0.05
+                        }
+                    }
+                },
+                'stemma': {
+                    'depth': depth,
+                    'width': {
+                        'law': 'Uniform',
+                        'min': 2,
+                        'max': width
+                    },
+                    'missing_manuscripts': {
+                        'law': 'Bernouilli',
+                        'rate': missing_rate
+                    }
+                }
+            }
+            # Rendu du template avec les données
+            output_from_parsed_template = template.render(data)
+            os.mkdir("/home/chausser/scenario1/")
```

Comment:
never have in main repo something that depends only on your system .... ! 
you need to build this so that the script create the folder in a relative fashion, or at the very least something that is configurable at the top of the script